instrumentation/trace/context.go

```diff
@@ -0,0 +1,50 @@
+package trace
+
+import (
+	"context"
+	"fmt"
+	"github.com/orbs-network/orbs-network-go/instrumentation/log"
+	"time"
+)
+
+type entryPointKeyType string
+
+const entryPointKey entryPointKeyType = "ep"
+
+type Context struct {
+	created   time.Time
+	name      string
+	requestId string
+}
+
+func NewContext(parent context.Context, name string) context.Context {
+	now := time.Now()
+	ep := &Context{
+		name:      name,
+		created:   now,
+		requestId: fmt.Sprintf("%s-%d", name, now.UnixNano()),
```

Comment:
I suggest we start using uuid v4 right away.

Reply:
why?

Reply:
because it's unique and does not rely on timestamps

Reply:
don't you think it's safe to assume that nano timestamps are also unique?

Reply:
| They might be reasonably unique, but we are going to propagate them between nodes.

It's a good point, but I'm not sure that we'll propagate request id between nodes. We'll discuss this on Thursday.

But - really, what are the odds of two transactions arriving in the same nanosecond anywhere on the node? also, we can add node id to the request id which *will* make it more unique.

The problem with `uuid`s, btw, is that it requires an external dependency and we're trying to limit them.

Reply:
i suggest we using something short for reuqest id.